python/xoscar/serialization/pyfury.py

```diff
@@ -3,15 +3,17 @@
 
 _fury = threading.local()
 _fury_not_installed = object()
-_register_class_list = set()
+_register_classes = set()
 
 
-def register_classes(*args):
+def register_class_to_fury(obj_type):
```

Comment:
This only register class for current thread, fury in other threads doens't get registerd. The deserialization may raise error if the register are not consisitent.

Reply:
Is this the root cause of random crash?

Reply:
Yes, another issue tis that the class register order must be consistent between serializatio and deserialization.

Is class registration constsitent always? Could you print all class registration too.